packages/editor/src/components/post-publish-panel/index.js

```diff
@@ -17,7 +17,7 @@ import {
 } from '@wordpress/components';
 import { withSelect, withDispatch } from '@wordpress/data';
 import { compose } from '@wordpress/compose';
-import { close } from '@wordpress/icons';
+import { closeSmall } from '@wordpress/icons';
```

Comment:
Should we replace all close icons or do you think there's a need for a small and a big one?

Reply:
I think there's a use case for both, honestly, it's okay they're big in the modal dialogs, but in the sidebar contexts they should not be quite as notable.